app/src/androidTest/assets/features/changeContactActivity.feature

```diff
@@ -0,0 +1,36 @@
+Feature: ContactPage
+  Editing a contact on the change contact page
+
+  Scenario Outline: Change the name of a contact
+      Given I am viewing the change contact activity
+      When I press the contact name
+      And I type a "<differentname>"
+      And I press on the save button
+      Then the contact should be updated
+      And I should go to the previous activity I visited
+
+      Examples:
+        | differentname  |
+        | OtherName  |
+        | 6j206j206j20  |
+        |   |
```

Comment:
Should this work? I think that an empty (or only spaces) username should not be valid...